subdomains/docs/_data/versions.json

```diff
@@ -1,24 +1,24 @@
 {
   "node": {
     "stable": {
-      "major": 20,
-      "full": "20.11.0"
+      "major": 22,
```

Comment:
The file is used for the command examples throughout the docs, e.g. the `volta install node@20.11.0` and similar blocks in [Managing your toolchain](https://docs.volta.sh/guide/understanding#managing-your-toolchain).

We don't _technically_ need to update it for a release, I just found it a useful time to make updates. to reflect the more recent versions of the tools so that the examples aren't using extremely old versions of e.g. Node 😄